src/shared/parse-helpers.h

```diff
@@ -23,3 +42,15 @@ int parse_socket_bind_item(
         int *ip_protocol,
         uint16_t *nr_ports,
         uint16_t *port_min);
+
+int open_file_parse(const char *v, OpenFile **ret);
+
+int open_file_check(const OpenFile *of);
+
+int open_file_to_string(const OpenFile *of, char **ret);
+
+OpenFile* open_file_free(OpenFile *of);
+DEFINE_TRIVIAL_CLEANUP_FUNC(OpenFile*, open_file_free);
+
+const char *open_file_flags_to_string(OpenFileFlags t) _const_;
+OpenFileFlags open_file_flags_from_string(const char *t) _pure_;
```

Comment:
maybe place this in a new .c/.h file pair src/shared/open-file.[ch] actually... it's quite a bunch of functions, I think it deserves its own .c file